log_config.py

```diff
@@ -0,0 +1,69 @@
+import json
+from datetime import datetime
+from logging import Formatter, config, getLogger
+
+
+class JsonFormatter(Formatter):
```

Comment:
suggestion: consider using off-the-shelf solution: https://github.com/nhairs/python-json-logger